ts/features/zendesk/store/actions/navigation.ts

```diff
@@ -1,9 +1,16 @@
 import { NavigationActions } from "react-navigation";
 import ZENDESK_ROUTES from "../../navigation/routes";
 
-export const navigateToZendeskAskPermissions = () =>
+export type navigateToZendeskAskPermissionsPayload = {
+  assistanceForPayment: boolean;
+};
+
+export const navigateToZendeskAskPermissions = (
+  params: navigateToZendeskAskPermissionsPayload
```

Comment:
```suggestion
  params: InferNavigationParams<typeof ZendeskAskPermissions>
```
This is the same approach used in similar other parts.
We should keep the code base homogeneous as far we can, until we want to follow that approach

obviously you have to change in 
`ts/features/zendesk/screens/ZendeskAskPermissions.tsx`

from 
`type Props = NavigationInjectedProps<navigateToZendeskAskPermissionsPayload>;`
to
`type Props = NavigationInjectedProps<{ assistanceForPayment: boolean }>;`

Reply:
You're right, thank you!
Refactored in [c00a9be](https://github.com/pagopa/io-app/pull/3593/commits/c00a9bec9bfdc356daeb8760233cba3dfc64cecb)